projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/update-email.e2e-spec.ts

```diff
@@ -2,29 +2,28 @@ import {
   updateEmailTest,
   verifyAsAnonymous,
 } from '../../../helpers/update-email';
-import { formats } from '../../../sample-data/viewports';
 
-describe('Update Email Address page', () => {
-  before(() =>
-    cy.window().then(win => {
-      win.sessionStorage.clear();
-    })
-  );
-
-  verifyAsAnonymous();
-  updateEmailTest();
-});
-
-describe(`${formats.mobile.width + 1}p resolution - Update Email page`, () => {
+describe('My Account - Update Email', () => {
   before(() => {
     cy.window().then(win => win.sessionStorage.clear());
-    cy.viewport(formats.mobile.width, formats.mobile.height);
   });
 
-  beforeEach(() => {
-    cy.viewport(formats.mobile.width, formats.mobile.height);
+  describe('update email test for anonymous user', () => {
+    verifyAsAnonymous();
   });
 
-  verifyAsAnonymous();
-  updateEmailTest();
+  describe('update email test for logged in user', () => {
+    beforeEach(() => {
```

Comment:
Please add in the email and password 'desktop' e2e test the following

before(() => {
      registerAndLogin();
      cy.reload();
      cy.visit('/');
    });

Looking at the other my account test, it will be consistent with the rest